### DISCUSSION TITLE
"Downloading APKINDEX.Tar.gz (0%)" never starts.

### DISCUSSION THREAD

**Discussion by antdude** [ish-app/ish#1375] [Q&A] [answered]
Hello.

I just downloaded and installed ish v1.1.1 into my old iPhone 6+ (iOS v12.5.2), but I can't seem to download APK stuff due to its "Downloading APKINDEX.Tar.gz (0%)". After a few minutes, it times out with:
"...
WARNING: Ignoring file:///ish/apk/main/x86/APKINDEX.tar.gz: IO ERROR
WARNING: Ignoring file:///ish/apk/community/x86/APKINDEX.tar.gz: No such
file or directory
..."

Why can't it download? Is it a bug? :( Thank you for reading and hopefully answering soon. :)

**Comment by 0x4248** [comment]
what speed is you connection? you can test it here www.speedtest.net

**Comment by antdude** [comment, marked as answer]
The developer fixed this issue in a newer version. I wonder why he never posted in here. Thanks.

**Comment by 0x4248** [comment]
Oh ok then can you make this comment as an answer to close the discussion thanks.

**Comment by antdude** [comment]
It doesn't let me close it. :/
